### ISSUE TITLE
Allow to complete available options

### ISSUE THREAD

**Issue by przepompownia** [phpactor/coc-phpactor#3]
Before a while I noticed that the indexer scans files within `./var` so I went to exclude it. Before I saw that I can pass configuration. It is good thing but still requires browsing the documentation. 

We could expose all available options in suggestions. It seems to be possible:
![image](https://user-images.githubusercontent.com/11404453/83449855-990a0400-a454-11ea-8233-9ee03b067e84.png)
https://github.com/neoclide/coc-tsserver/blob/master/package.json#L142

Exposing all options (reasonable in this context) allow make it more user-friendly and show the power of whole tool. 

I wanted to make PR with any working example but I did not found how to add `coc-phpactor` as a node module with sources within `extensions` directory.

**Comment by dantleech** [comment]
Yes - we can do that - but - the problem is that there are [lots of them](https://phpactor.readthedocs.io/en/develop/reference/configuration.html). 

I would only add the most common options here (e.g. the include/exclude paths etc). We can now also generate the schema (the documentation is generated). But that requires more thought again.

**Comment by dantleech** [comment]
Apparently you can add the extension to the RTP and it should just work (I haven't tried it, I actually just edited it in the extensions directory and copied it to the repo :eyes: ).

For vscode I just symlinked the repository to `.vscode/extensions` - maybe something similar also works.

**Comment by dantleech** [comment]
Phpactor now ships with `phpactor.schema.json` which could be used to generate the configuration here, the problem would be ensuring that it remains in sync

**Comment by przepompownia** [comment]
I dropped using CoC about year ago and no one else is interested in it here so let's close this issue.

**Comment by dantleech** [comment]
I think it's still valid if somebody wanted to pick up maintainence of this
